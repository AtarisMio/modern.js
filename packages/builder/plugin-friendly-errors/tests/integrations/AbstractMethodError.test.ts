import CommentCompilationWarning from 'webpack5/lib/CommentCompilationWarning';
import { describe, it, expect } from 'vitest';

describe('CommentCompilationWarning', () => {
  it('should print raw error', () => {
    const error = new CommentCompilationWarning('foo', {
      start: { line: 1, column: 10 },
      end: { line: 3, column: 20 },
      index: 2,
    });
    expect(() => {
      throw error;
    }).toThrowErrorMatchingSnapshot();
  });
});
