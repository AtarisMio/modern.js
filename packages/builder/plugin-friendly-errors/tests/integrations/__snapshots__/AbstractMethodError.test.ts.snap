// Vitest Snapshot v1

exports[`CommentCompilationWarning > should print raw error 1`] = `"foo"`;
